src/views/CooperationInfo.vue

```diff
@@ -254,25 +403,29 @@ export default defineComponent({
     DataTable,
     Column,
     Menu,
+    ConfirmPopup,
   },
   data() {
     return {
-      houseActions: [
-        {
-          label: 'Видалити',
-          icon: 'pi pi-times',
-          command: () => {
-            this.confirmDeleteHouse();
+      houseActions: (house: HouseInterface) => {
```

Comment:
Why do you need house argument if you never use it in function. By the way, you can put an array in houseActions, not the function that returns an array

Reply:
I need "house" argument for showSuccessEdit(house) inside houseActions, so i guess I will keep function with argument

Reply:
Yep, i missed that line of code, my bad